### PR TITLE
Update application.json

### PR DESCRIPTION
**Pull request by splatterxl**
👀

### CODE REVIEW

File: flags/application.json [hunk 1]
```diff
@@ -14,6 +14,26 @@
...
+    "shift": 8,
+    "undocumented": true
+  },
+  "ALLOW_ACTIVITY_ACTION_SPECTATE": {
+    "description": "Whether the application can allow users to spectate a game from RPC",
+    "shift": 9,
+    "undocumented": true
+  },
+  "ALLOW_ACTIVITY_ACTION_JOIN_REQUEST": {
+    "description": "Allows the application can allow users to request to join a user's game from RPC",
```
Comment (MeguminSama) [inline flags/application.json:33]: Poor wording - possibly:
`"Enables activity join requests for the application",`

File: flags/application.json [hunk 2]
```diff
@@ -14,6 +14,26 @@
...
+    "shift": 5,
+    "undocumented": true
+  },
+  "ALLOW_ASSETS": {
+    "description": "Relates to RPC application assets",
+    "shift": 8,
+    "undocumented": true
+  },
+  "ALLOW_ACTIVITY_ACTION_SPECTATE": {
+    "description": "Whether the application can allow users to spectate a game from RPC",
```
Comment (MeguminSama) [inline flags/application.json:28]: For consistency with my prior comment:
`"Enables activity spectating for the application",`

File: flags/application.json [hunk 3]
```diff
@@ -14,6 +14,26 @@
     "shift": 4,
     "undocumented": true
   },
+  "RPC_PRIVATE_BETA": {
+    "description": "Application has access to the RPC HTTP server",
+    "shift": 5,
+    "undocumented": true
+  },
+  "ALLOW_ASSETS": {
+    "description": "Relates to RPC application assets",
```
Comment (MeguminSama) [inline flags/application.json:23]: `"description": "Allow the creation of RPC assets",`

File: flags/application.json [hunk 4]
```diff
@@ -14,6 +14,26 @@
     "shift": 4,
     "undocumented": true
   },
+  "RPC_PRIVATE_BETA": {
+    "description": "Application has access to the RPC HTTP server",
```
Comment (MeguminSama) [inline flags/application.json:18]: Possibly replace `RPC HTTP server` with `RPC API` ?

File: flags/application.json (hunk 2 above)
Comment (dolfies) [inline flags/application.json:28]:
```suggestion
    "description": "Whether the application can have activities that are spectable",
```

**Review by MeguminSama — CHANGES_REQUESTED** [review]

File: flags/application.json (hunk 1 above)
Comment (dolfies) [inline flags/application.json:33]:
```suggestion
    "description": "Allows the application can have activities that require a request to join",
```

File: flags/application.json (hunk 4 above)
Comment (splatterxl) [reply flags/application.json:18]: Apps can still interact with the RPC API using the local IPC server, the HTTP server hosted by the desktop client is locked to the beta though

Comment (dolfies) [inline flags/application.json:18]:
```suggestion
    "description": "Whether the application is in the RPC private beta",
```

File: flags/application.json (hunk 3 above)
Comment (dolfies) [inline flags/application.json:23]:
```suggestion
    "description": "Whether the application can create RPC assets",
```

**Review by dolfies — CHANGES_REQUESTED** [review]
Some wording nits

File: flags/application.json (hunk 4 above)
Comment (MeguminSama) [reply flags/application.json]: RPC API has more functionality than IPC

Comment (dolfies) [reply flags/application.json]: RPC != IPC

**Review by dolfies — APPROVED** [review]

**Review by MeguminSama — APPROVED** [review]

**Pull request merged**